client/src/components/select-formula-view/SelectFormulaForm.js

```diff
@@ -114,12 +167,12 @@ const SelectFormulaForm = ({ assignments, formulas, courseId }) => {
           flexWrap: 'wrap',
           justifyContent: 'space-between'
         }}>
-          <Typography width={350} m={3}>Specify attribute values for the sub study attainnments</Typography>
+          <Typography width={350} m={3}>Specify attribute values for the sub study attainments</Typography>
```

Comment:
With my screen max width, if the with of the typography is 350, the buttons will displayed under it. I tested that with the width 320, the buttons will be displayed next to it and the format of the typography doesn't change, so maybe that could be changed.